src/builtins/BuiltinTemporal.cpp

```diff
@@ -29,6 +29,14 @@
 namespace Escargot {
 #if defined(ESCARGOT_ENABLE_TEMPORAL)
 
+#define CHECK_TEMPORAL_OBJECT_HAS_YEAR_AND_MONTH(temporalDateLike) \
+    !(temporalDateLike.isObject() && (temporalDateLike.asObject()->isTemporalPlainDateObject() || temporalDateLike.asObject()->isTemporalPlainDateTimeObject() || temporalDateLike.asObject()->isTemporalPlainYearMonthObject()))
+
+#define CHECK_TEMPORAL_PLAIN_YEAR_MONTH(state, value)                                  \
```

Comment:
Inline functions are used for these before. Why changing these?

Reply:
@clover2123 requested it https://github.com/Samsung/escargot/pull/1122#discussion_r937347504

Reply:
Ok. Maybe the other simple inlines could be changed to macro then (other patch of course)